### DISCUSSION TITLE
Gateway zigbee remoto

### DISCUSSION THREAD

**Discussion by essegi65** [indomus/forum#115] [HUB: Home Assistant]
Salve a tutti, ho un RPI 4 dedicato con Hassio, vorrei incominciare a inserire il protocollo zigbee per ridurre i device sul mio wifi. Siccome ho il RPI4 in una cantina collegato al modem tramite cavo di rete non potei mettere una chiavetta al RPI per problemi di distanze con il segnale zigbee (non avrei nemmeno la possibilità di mettere dei device che fanno da ripetitori nel tragitto cantina - modem), sapete indicarmi se esiste un gateway da mettere magari vicino al modem e collegarlo via cavo di rete o wifi al mio RPI4?
Grazie in anticipo a tutti.

**Comment by snakuzzo** [comment]
Non sei assolutamente vincolato al RPi...
Considera che le soluzioni al tuo problema sono diverse, però ti consiglierei di dare una letta al nostro articolo sul [Z-Bee Duo](https://indomus.it/componenti/z-bee-duo-zigbee-coordinator-usb-hat-per-raspberry-pi-zero/) realizzato da @Gio-dot

**Reply by gio-dot** [reply]
In più rispetto a quanto detto da Diego, sto lavorando ad un piccolo accessorio da mettere nella mia chiavetta che permette di fare un ponte wifi (ser2net) rendendola così un gateway zigbee staccato da home assistant. È una soluzione più semplice ed economica rispetto a quella che prevede l'utilizzo di un Raspberry pi Zero abbinato sempre alla mia chiavetta.
Questo accessorio sarà disponibile in un paio di settimane.

**Comment by essegi65** [comment]
Grazie a tutti e due, mi metto a verificare
